config/config.go

```diff
@@ -172,6 +167,10 @@ func (c *Config) parseConfig(ver protocolVersion) error {
 	if err := protoVersionCheck(ver); err != nil {
 		return err
 	}
+	if exists := c.v.Get(fmt.Sprintf("server%d", ver)); exists == nil {
```

Comment:
Why are you moving this? Was it returning an error before? Wasn't it returning nil in line 185 after getting a nil listenAddr?

Reply:
It was returning an error previously when the config file only had v4 configuration, because it would fail out before reaching line 185:

```
INFO[2019-04-22T11:45:30+01:00] Registering plugin "file"                    
INFO[2019-04-22T11:45:30+01:00] Registering plugin "server_id"               
INFO[2019-04-22T11:45:30+01:00] Loading configuration                        
FATA[2019-04-22T11:45:30+01:00] error parsing config: dhcpv6: missing `server6.interface` directive 
```

```
# config.yml
server4:
    listen: '0.0.0.0:67'
    interface: "eth0"
    plugins:
        - server_id: 192.168.1.12
```